.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thank you for submitting this issue.'
```

Comment:
Can we add something about pull requests are welcome or something like this? `Thanks for opening your first issue. Pull requests are always welcome!` We want the OSS world to know that we are open to PRs.

Reply:
I'll change that now! :)